learn-css-grid-by-building-a-magazine/css/style020.css

```diff
@@ -5,45 +5,8 @@
   margin: 0;
 }
 
-html {
-  box-sizing: border-box;
-  font-size: 62.5%;
-}
-
-body {
-  font-family: "Baskervville", serif;
-  color: linen;
-  background-color: #202020;
-}
-
-h1 {
-  font-family: "Anton", serif;
-}
-
-h2,
-h3,
-h4,
-h5,
-h6 {
-  font-family: "Raleway", sans-serif;
-}
-
-a:link,
-a:visited {
-  text-decoration: none;
-  color: linen;
-}
-
-section:last-child {
-  margin-bottom: 3rem;
-}
-
-/* To make you columns responsive on any devices, you could use minmax function. The value of minmax function could be `<length>`, `<percentage>`, a `<flex>` value or one of the keyword values `max-content`, `min-content`, or `auto`. 
-For example: `minmax(200px, 1fr) minmax(min-content, 500px)`.
-The value that you write before is the max value, now you could wrap it with minmax function and give the `min` value to it. 
-Wrap each of the already defined values of your `grid-template-columns` property with minmax function and set the min value to `2rem`, `min-content` and `2rem` respectiveley. 
-*/
-main {
-  display: grid;
-  grid-template-columns: 1fr 94rem 1fr;
-}
\ No newline at end of file
+/* 
```

Comment:
```
It's best to set typography on the `html` and `body` elements.
First, target the `html` element and set its `font-size` to `62.5%`.
Why set it to this value? The default font-size is 16px, so here we're effectively making it 10px (62.5% * 16px = 10px).
This will make it easier for us to work with `rem` units later, as `2rem` will equal 20px, `3rem` 30px, and so on.
```